update repository URLs (#897)
helm/helm-www: content/en/docs/faq/_index.md

## Troubleshooting

### I am getting a warning about "Unable to get an update from the "stable" chart repository"

> Run `helm repo list`. If it shows your `stable` repository pointing to a Google link, you
> will need to update that repository. On November 13, 2020, Google terminated their copy
> of the Helm stable chart repository. The current version is available at
> `https://charts.helm.sh/stable`. You can run the following command to fix your repository:

Reviewer: ```suggestion
Run `helm repo list`. If it shows your `stable` repository pointing to a `storage.googleapis.com` URL, you
will need to update that repository. On November 13, 2020, the Helm Charts repo [became unsupported](https://github.com/helm/charts#deprecation-timeline) after a year-long deprecation. An archive has been made available at
`https://charts.helm.sh/stable` but will no longer receive updates. 

You can run the following command to fix your repository:
```

```console
$ helm repo add stable https://charts.helm.sh/stable --force-update  
```

> The same goes for the `incubator` repository, which Google also discontinued support for.

Reviewer: ```suggestion
The same goes for the `incubator` repository, which has an archive available at https://charts.helm.sh/incubator.
```

You can run the following command to repair it:

```console
$ helm repo add incubator https://charts.helm.sh/incubator --force-update  
```

### I am getting the warning 'WARNING: "kubernetes-charts.storage.googleapis.com" is deprecated for "stable" and will be deleted Nov. 13, 2020.'

The old Google helm chart repository has been replaced by a new Helm chart repository.

Run the following command to permanently fix this:

```console
$ helm repo add stable https://charts.helm.sh/stable --force-update  
```

If you get a similar error for `incubator`, run this command:

```console
$ helm repo add incubator https://charts.helm.sh/incubator --force-update  
```

### When I add a Helm repo, I get the error 'Error: Repo "https://kubernetes-charts.storage.googleapis.com" is no longer available'

> 
> The old Helm chart repositories that were once considered official have been removed by
> Google when their support period expired. The new URL for these repositories has changed
> to `https://charts.helm.sh/stable` and `https://charts.helm.sh/incubator`. The command
> `helm repo add` will not let you add the old URLs unless you specify `--use-deprecated-repos`.

Reviewer: ```suggestion

The Helm Chart repositories are no longer supported after [a year-long deprecation period](https://github.com/helm/charts#deprecation-timeline). 
Archives for these repositories are available at `https://charts.helm.sh/stable` and `https://charts.helm.sh/incubator`, however they will no longer receive updates. The command
`helm repo add` will not let you add the old URLs unless you specify `--use-deprecated-repos`.
```
